### PR TITLE
Changes codecov, to remove Docker

### PR DESCRIPTION
**Pull request by Marauderer97**
Refer to #161, #250 
Please donot merge as yet, 
Need to make sure travis first runs without failing,
This PR attempts to provide the repo with a code coverage by temporarily removing Docker, to first let codecov work without it.

### CODE REVIEW

**Comment by Marauderer97** [comment]
Travis compiled, so if we merge this, codecov will be available but without comments facility. Can we merge this as the first step, so that we atleast have some codecov, and later work on the Docker issue being faced in #161, since that has hit a road block.

**Comment by mariobehling** [comment]
@Marauderer97 Please ask and assign your co-developers to review.

**Comment by Marauderer97** [comment]
@harshit98 @nikhilrayaprolu Please review. :)

**Comment by harshit98** [comment]
@Marauderer97 It is like, we are reverting back the changes. Yes, we can try other options if you're having.

File: codecov.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 codecov:
-  branch: gh-pages1
+  branch: gh-pages
```
Comment (nikhilrayaprolu) [inline codecov.yml]: are you sure that it should be gh-pages or should it be master?

**Pull request merged**